### PR TITLE
Prompt for options

### LINKED ISSUE #84: Prompt for platform when no platform passed to CLI
Currently it shows an error, will be nice to have it ask the platform to avoid extra step

### PR DESCRIPTION
**Pull request by grabbou**
Fixes #84

### CODE REVIEW

**Comment by grabbou** [comment]
TODO: Figure out how to display `option.note` below the question and hint how to use it next time with value provided.

File: src/cliEntry.js [hunk 1]
```diff
@@ -38,48 +39,50 @@ const NOT_SUPPORTED_COMMANDS = [
...
+
+    const parse = typeof option.parse === 'function'
+      ? option.parse
+      : val => val;
+
+    let value = flags[option.name] ? parse(flags[option.name]) : defaultValue;
+
+    if (
+      // If value is required but not provided
+      (option.required && !value) ||
```
Comment (satya164) [inline src/cliEntry.js]: What if `value` is a boolean? `!value` will match `!true`

**Comment by grabbou** [comment]
Makes sense. I'll make it to detect undefined case instead.

**Comment by grabbou** [comment]
@satya164 this is ready for another review.

**Pull request merged**